### PR TITLE
Update hu-hu.json

### PR DESCRIPTION
**Pull request by summoner001**
Correcting translation.

### CODE REVIEW

File: src/lang/hu-hu.json [hunk 1]
```diff
@@ -388,7 +388,7 @@
 	"delete entries": "Biztosan töröl {count} elemet?",
 	"deleting items": "{count} elem törlése…",
 	"import project zip": "Projekt importálása zip-ből",
-	"changelog": "Változásnapló",
+	"changelog": "Változáslista",
```
Comment (bajrangCoder) [inline src/lang/hu-hu.json:391]: Why change list , instead of change log

Comment (bajrangCoder) [reply src/lang/hu-hu.json:391]: Previous translation is correct

Comment (summoner001) [reply src/lang/hu-hu.json:391]: We have a software translation dictionary https://bkil.github.io/openscope-dict-eng-hun/#Changelog

Changelog -> is a file
Changelist -> what the file contain

https://keepachangelog.com/en/1.1.0/

What is a changelog?

A changelog is a file which contains a curated, chronologically ordered list of notable changes for each version of a project.

Comment (bajrangCoder) [reply src/lang/hu-hu.json:391]: Thanks for clarifying it

**Pull request merged**